ydb/core/kqp/ut/scheme/kqp_scheme_ut.cpp

```diff
@@ -3837,6 +3837,45 @@ Y_UNIT_TEST_SUITE(KqpScheme) {
         }
     }
 
+    Y_UNIT_TEST(ModifyPermissionsByRelativePath) {
+        TKikimrRunner kikimr;
+        auto db = kikimr.GetTableClient();
+        auto session = db.CreateSession().GetValueSync().GetSession();
+
+        {
+            {
+                const TString query = R"(
+                    CREATE TABLE `MyApp/Orders` (
+                        id Int32 NOT NULL,
+                        value Int32,
+                        PRIMARY KEY (id)
+                    );
+                )";
+
+                auto result = session.ExecuteSchemeQuery(query).GetValueSync();
+                UNIT_ASSERT_C(result.IsSuccess(), result.GetIssues().ToString());
+            }
+
+            {
+                const TString query = R"(
+                    GRANT SELECT ON `MyApp/Orders` TO ydbuser;
+                )";
+
+                auto result = session.ExecuteSchemeQuery(query).GetValueSync();
+                UNIT_ASSERT_C(result.IsSuccess(), result.GetIssues().ToString());
+            }
+
+            {
+                const TString query = R"(
+                    REVOKE SELECT ON `MyApp/Orders` FROM ydbuser;
+                )";
+
+                auto result = session.ExecuteSchemeQuery(query).GetValueSync();
+                UNIT_ASSERT_C(result.IsSuccess(), result.GetIssues().ToString());
+            }
+        }
```

Comment:
Could you please add the same test but with the query client? See the links below for details:

https://github.com/ydb-platform/ydb/blob/b07b2ad5759bec15e8dd38b321c4fb468177214e/ydb/core/kqp/ut/olap/delete_ut.cpp#L13
https://github.com/ydb-platform/ydb/blob/b07b2ad5759bec15e8dd38b321c4fb468177214e/ydb/core/kqp/ut/olap/delete_ut.cpp#L35C38-L35C44

generally PR is ok from my side!

Reply:
Done